bool moving
geometry_msgs/Pose pose
float64[] q
